apps/public-docsite-resources/config/api-docs.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
   apiJsonPaths: [
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29292&commitId=310fbf3ba58a3c9fabd67769f938757fb20b715b&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-TreeChart</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-TreeChart.Compact.default.chromium.png|91385| Changed|
|react-charting-TreeChart.Compact - RTL.default.chromium.png|91288| Changed|
|react-charting-TreeChart.Basic.default.chromium.png|91059| Changed|
|react-charting-TreeChart.Basic - Dark Mode.default.chromium.png|109565| Changed|
|react-charting-TreeChart.Basic - RTL.default.chromium.png|90986| Changed|
|react-charting-TreeChart.Compact - Dark Mode.default.chromium.png|115228| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>